src/ansys/rocky/core/launcher.py

```diff
@@ -21,22 +21,22 @@
 # SOFTWARE.
 """Module that exposes functions to launch a Rocky application session."""
 import contextlib
-import os
 from pathlib import Path
 import subprocess
 import time
-from typing import Optional, Union
 
 from Pyro5.errors import CommunicationError
+from ansys.tools.path import get_available_ansys_installations
 
 from ansys.rocky.core.client import DEFAULT_SERVER_PORT, RockyClient, connect_to_rocky
 from ansys.rocky.core.exceptions import RockyLaunchError
 
 _CONNECT_TO_SERVER_TIMEOUT = 60
+MINIMUM_ANSYS_VERSION_SUPPORTED = 242
 
 
 def launch_rocky(
-    rocky_exe: Optional[Union[Path, str]] = None,
+    rocky_version: str | int = None,
```

Comment:
We don't want to break this signature and make it "less flexible". People that uses Rocky from the standalone installer could install Rocky in non-standard places. Please keep `rocky_exe` parameter.

We can add an optional parameter named `ansys_version` where user can set a predefined version. When `ansys_version` is given, we must be sure that user didn't give `rocky_exe`.

Reply:
Done, please let me know if this is good enough.